src/puppetlabs/puppetdb/command/dlo.clj

```diff
@@ -49,10 +49,10 @@
   (with-open [path-stream (Files/newDirectoryStream path)]
     (doseq [path path-stream]
       ;; Assume the trailing .json here and in entry-cmd-err-filename below.
-      (when-let [cmd (:command
-                      (and (.endsWith path ".json")
-                           (parse-cmd-filename (-> path .getFileName str))))]
-        (update-metrics metrics cmd (Files/size path))))))
+      (let [name (-> path .getFileName str)]
+        (when-let [cmd (:command (and (.endsWith name ".json")
+                                      (parse-cmd-filename name)))]
```

Comment:
parse-cmd-filename has a regex bug that will prevent this from working -- [0-9+] needs to be swapped with [0-9]+.